libs/domain/product/src/services/adapter/normalizers/concrete-products/concrete-products.normalizer.ts

```diff
@@ -27,7 +27,12 @@ export function concreteProductsNormalizer(
             ProductNormalizer
           ),
           transformer.transform(abstract[categoryKey], CategoryIdNormalizer),
-        ]).pipe(map(([product, nodeId]) => ({ ...product, ...nodeId })))
+        ]).pipe(
+          map(([product, nodeId]) => ({
+            ...product,
+            ...nodeId,
+          }))
+        )
```

Comment:
Potential BC. Would the consuming code still work if a custom normalizer is used? Could we add this in a feature flag?

Reply:
I didn't add there something, prettier just formatted it differently